index.html

```diff
@@ -1,51 +1,72 @@
 <!DOCTYPE html>
-<title>WACZ Uploader</title>
-<h1>WACZ Uploader</h1>
-<style>
-  html {
-    background: #FAFAFA;
-  }
+<html>
```

Comment:
This is defs a bikeshedding/style thing, but would you be into removing the html/head/body elements to reduce the amount on indentation in the file?

In html5 they're optional so I've taken to omitting them. :x

Reply:
We should keep `html` in case we want to support languages (I meant to add `lang=en`) and the head tag is needed to wrap the meta and title tags.

My editor is set to auto-indent HTML so I can disable that to reduce indentations--I'll update this file in the PR, too.

HTML specs for reference:
- https://html.spec.whatwg.org/dev/semantics.html#semantics
- https://html.spec.whatwg.org/dev/sections.html#the-body-element